tests/StackExchange.Redis.Tests/Sets.cs

```diff
@@ -11,14 +11,52 @@ public class Sets : TestBase
     {
         public Sets(ITestOutputHelper output, SharedConnectionFixture fixture) : base (output, fixture) { }
 
+        [Fact]
+        public void SetIntersectionLength()
+        {
+            using var conn = Create();
+            Skip.IfBelow(conn, RedisFeatures.v7_0_0_rc1);
```

Comment:
This one needs a main merge so we run the tests (see #2079 just merged - this lights up 7.0-rc3 image to run instead of skip these!)